src/emit/solana/target.rs

```diff
@@ -2399,4 +2433,24 @@ impl<'a> TargetRuntime<'a> for SolanaTarget {
             .builder
             .build_return(Some(&binary.return_values[&ReturnCode::Success]));
     }
+
+    fn log_runtime_error(
+        &self,
+        bin: &Binary,
+        reason_string: String,
+        reason_loc: Option<Loc>,
+        ns: &Namespace,
+    ) {
+        if !bin.options.log_runtime_errors {
+            return;
+        }
+        emit_context!(bin);
```

Comment:
This macro should only be used in Substrate. It is not defined for Solana.

Reply:
sorry, silly copy and paste mistake